domain.yml

```diff
@@ -332,14 +330,14 @@ templates:
   utter_no_guide_for_switch:
   - text: Sorry, but we don't have a guide for this yet.
   utter_ask_which_product:
-  - text: Ok, which product would you like to know more about? Rasa (Core + NLU)
-      or Rasa X?
+  - text: Ok, which product would you like to know more about? Rasa (Core + NLU) or
+      Rasa X?
```

Comment:
The existence of this utterance is all the more reason for keeping the other one as "would you like to" instead of "which one"

Reply:
Could you elaborate why "this utterance is all the more reason for keeping the other one as "would you like to" instead of "which one""?